client/src/main/com/sinch/sdk/domains/verification/models/requests/VerificationReportFlashCallRequestParameters.java

```diff
@@ -0,0 +1,56 @@
+package com.sinch.sdk.domains.verification.models.requests;
+
+import com.sinch.sdk.domains.verification.models.VerificationMethodType;
+
+public class VerificationReportFlashCallRequestParameters
+    extends VerificationReportRequestParameters {
+
+  private final String cli;
+
+  public VerificationReportFlashCallRequestParameters(String cli) {
```

Comment:
The famous `cli` that we will never know about 🙃

Reply:
Right! 🤔